.packit.yaml

```diff
@@ -77,3 +77,20 @@ jobs:
     packages: [python-podman-fedora]
     dist_git_branches:
       - fedora-branched # rawhide updates are created automatically
+
+  # Test linting on the codebase
+  # This test might break based on the OS and lint used, so we follow fedora-latest as a reference
+  - job: tests
+    trigger: pull_request
```

Comment:
include `packages: [python-podman-fedora]` here and likewise `-centos` for centos jobs. That will avoid the test jobs that are on fedora environment but say `python-podman-centos` in the end and vice versa.

Reply:
hmm, this comment would probably be relevant to the test job below and not this one.